infrastructure/smart-contracts/test/Referee.mjs

```diff
@@ -566,11 +620,40 @@ export function RefereeTests(deployInfrastructure) {
 				currentAssertion,
 				previousAssertion
 			);
+
 			for (let i = 0; i < res.confirmData.length; i++) {
 				assert.equal(res.confirmData[i], res.nodes[i][2], "confirmData mismatch");
 			}
 		});
 
+		it("Check failure to confirm node with wrong blockhash and sendroot", async function () {
```

Comment:
Please help me understand what this testcase is testing ?
The flow is, NodeCreated -> NodeConfirmed -> Challenge Submitted.

Reply:
same comment as above